package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "sentiment-chat",
+  "name": "hoal",
```

Comment:
What? LOL

Reply:
Hahaha sorry. It was a name "hola" (misspelled). I already fixed the name.